vyper/ast_utils.py

```diff
@@ -87,17 +76,33 @@ def parse_python_ast(source_code: str,
     if isinstance(node, list):
         return _build_vyper_ast_list(source_code, node, source_id)
     elif isinstance(node, python_ast.AST):
-        class_name = node.__class__.__name__
-        if hasattr(vyper_ast, class_name):
-            vyper_class = getattr(vyper_ast, class_name)
-            init_kwargs = _build_vyper_ast_init_kwargs(
-                source_code, node, vyper_class, class_name, source_id
-            )
-            return vyper_class(**init_kwargs)
+        # necessary for python3.8 compatibility
+        if isinstance(node, python_ast.Num):
+            class_name = "Num"
+        elif isinstance(node, python_ast.Str):
+            class_name = "Str"
+        elif isinstance(node, python_ast.Bytes):
+            class_name = "Bytes"
+        elif isinstance(node, python_ast.NameConstant):
+            class_name = "NameConstant"
         else:
+            class_name = node.__class__.__name__
```

Comment:
Not sure I understand the fix here, as far as I can tell all these types are replaced by Constant(value=) now, doesn't that mean we have to translate each constant type and port over the value attribute (e.g. `.s` for Str and `.n` for number) ?

Reply:
And would that mean something the replacement step would have to map something like `isinstance(node.value, str)` becomes 'vyper_ast.Str(s=node.value)`

Reply:
See new commits based on our discussion :+1: